src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.WebSearch/plugin.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ID": "9F1B49201C3F4BF781CAAD5CD88EA4DC",
+  "ActionKeyword": "#",
```

Comment:
this should be "?", right?

Reply:
I changed it because another plugin uses "?" by default. forgot to update the pr

Reply:
Indexer uses ? I think. I like ??

Reply:
windows search plugin.

i think using one char is better

Reply:
ok ill change it